### PR TITLE
#296: [Panel] Panel should have shrink=false for all its children (closes #296)

### LINKED ISSUE #296: [Panel] Panel should have shrink=false for all its children
The last change about `overflow: hidden;` for `panel-header` broke the PanelMenu visibility in open state.
The alternative solution could be set `shrink={false}` for `PanelHeader` and its inside element (`.panel-header-icon`, `.panel-header-title`, `.panel-header-content`) to make it take a `min-width` equals to its inside width.

@buildo/frontend thoughts?

### PR DESCRIPTION
**Pull request by ascariandrea**
Issue #296

### CODE REVIEW

File: src/Panel/panel.scss [hunk 1]
```diff
@@ -60,12 +60,12 @@
   }
 
   > .panel-header {
-    overflow: hidden;
     color: $white;
     background-color: $charcoal;
 
     .expand-icon {
       font-size: 20px;
+      padding: 0 10px;
```
Comment (FrancescoCioria) [inline src/Panel/panel.scss]: Why? It already has `basis={30}` which is more than the width of the icon

File: src/Panel/PanelHeader.js [hunk 2]
```diff
@@ -58,7 +58,8 @@ export default class PanelHeader extends React.Component {
           hAlignContent='center'
           style={{ cursor: 'pointer' }}
           className='panel-header-icon'
-          flexBasis={30}
+          basis={30}
+          shrink={false}
```
Comment (FrancescoCioria) [inline src/Panel/PanelHeader.js]: This can be removed as shrink is already false when you set a basis

Comment (ascariandrea) [inline src/Panel/PanelHeader.js]: :+1:

File: src/Panel/panel.scss (hunk 1 above)
Comment (ascariandrea) [inline src/Panel/panel.scss]: I used padding before change `flexBasis` to `basis` and I forgot to remove it :see_no_evil:

**Comment by FrancescoCioria** [comment]
👌

**Comment by ascariandrea** [comment]
To be rebased after #302 merge.

**Comment by FrancescoCioria** [comment]
merged #302, please rebase

**Comment by ascariandrea** [comment]
Done :+1:

**Pull request merged**